llvm-external-projects/iree-dialects/lib/Dialect/LinalgTransform/IR/StructuredTransformOpsExt.cpp

```diff
@@ -1223,10 +1223,14 @@ testMatchCallbackCallback(transform_ext::MatchCallbackResult &res, Location loc,
 ///   - the "fill" op preceding the reduction;
 ///   - reduction op;
 ///   - trailing elementwise op, if any.
+///
+/// The `reductionCallback` version additionally checks that the matched
+/// operations are the only ones implementing TileableInterface within the
+/// parent function. The `reductionPartialCallback` does not.
```

Comment:
maybe add a sentence about slicing IR from a larger graph into a dispatch region vs performing codegen of exactly all the things previously sliced into a dispatch region: we reuse the same code for both.